pkgs/applications/science/logic/uhdm/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "UHDM";
+  version = "2022.05.15";
```

Comment:
There is now a tag [v0.9.1.37](https://github.com/chipsalliance/UHDM/releases/tag/v0.9.1.37) in UHDM.

So we can have here a proper version
```nix
version = "0.9.1.37";
# and below
src = ... {
    ref = "v${version}";
};
```